src/components/Table/EditColumnsButton.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { noop } from "src/utils";
+import { click, render } from "src/utils/rtl";
+import { EditColumnsButton } from "./EditColumnsButton";
+import { GridColumn } from "./GridTable";
+import { SimpleHeaderAndData } from "./simpleHelpers";
+
+type Data = { name: string | undefined; value: number | undefined };
+type Row = SimpleHeaderAndData<Data>;
+const nameColumn: GridColumn<Row> = { name: "name", header: "Name", data: ({ name }) => name };
+const valueColumn: GridColumn<Row> = { name: "value", header: "Value", canHide: true, data: ({ value }) => value };
+const actionColumn: GridColumn<Row> = {
+  name: "actions",
+  header: "Action",
+  canHide: true,
+  data: () => <div>Actions</div>,
+};
+
+describe("EditColumnsButton", () => {
+  it("Should render editColumnsButton", async () => {
+    // Given an edit columns button
+    const r = await render(
+      <EditColumnsButton
+        trigger={{ label: "Columns" }}
+        columns={[nameColumn, valueColumn, actionColumn]}
+        setColumns={noop}
+      />,
+    );
+    // Expect to render the button with the expected label
+    expect(r.columns().textContent).toBe("Columns");
```

Comment:
**nit/request:** Please don't use `Expect` in the comments, as it is redundant with the following line of `expect(...)`. Typically we use `Given / When /Then` comments, so this would be something like:
```
// Then the button renders with the correct label
```
(There are more `// Expect ...` comments like this in the rest of the tests. Could you update those too, please?)